app/assets/stylesheets/style.scss

```diff
@@ -1,32 +1,33 @@
 body {
-  margin:0px;
   background-image:
   url("http://dfwpartyplanners.com/wp-content/uploads/2014/09/party.jpg");
+  margin: 0px;
 }
-
 .calendar {
   td {
-    width:150px;
-    max-width: 150px;
-    height:100px;
+    height: 100px;
     max-height: 100px;
+    max-width: 150px;
+    width: 150px;
   }
 }
-.scrollable{
-  width:150px;
-  max-width: 150px;
-  height:80px;
+.scrollable {
+  height: 80px;
+  max-width: 140px;
   max-height: 80px;
-  overflow-y: scroll;
-  color: blue;
+  overflow-y: auto;
+  width: 140px;
+}
+#event_show {
+  margin-top: 50px;
+  padding-top: 10px;
 }
-
 #map {
+  background-color:#000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space